packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1262,4 +1293,19 @@ module.exports.external = [
       return undefined; 
     },
   },
+  {
+    name: "concatDefined",
+    description: "Concatenates defined objects, checking for null, undefined, or UNK nullFlavor.",
```

Comment:
used for generating IDs for resources. Ensures that we dont include undefined values in IDs for resources, which helps a lot with deduplicating. So now 
{name: John Doe}, {telcom: nullFlavor} == {name: John Doe}